README.md

```diff
@@ -84,13 +84,27 @@ The CLI is available by using the command `jsipfs` in your terminal. This is ali
 
 The code published to npm that gets loaded on require is in fact a ES5 transpiled version with the right shims added. This means that you can require it and use with your favourite bundler without having to adjust the asset management process.
 
+```js
+var ipfs = require('ipfs');
+```
+
 ### Use in a browser using a script tag
 
-Simply include ipfs in your bundled javascript file.
+Loading this module in a browser (using a `<script>` tag) makes the `Ipfs` object available in the global namespace.
 
-```html
-<script type="text/javascript" src="assets/bundled.js"></script>
-```
+The last published version of the package becomes available on [npmcdn](https://npmcdn.com/) and thus you may use it as the source:
```

Comment:
Just to confirm: do you suggest to link the word “npmcdn” to https://npmcdn.com/ipfs/dist/ instead of the npmcdn's homepage?

Originally I intended to use the examples (`<script src="https://npmcdn.com/ipfs/dist/index.min.js">` and `<script src="https://npmcdn.com/ipfs/dist/index.js">` below) to explain the script's location but I left the word “npmcdn” to tell the ignorant public what npmcdn is. That's how the README.md of `ipfs-api` [does it currently](https://github.com/ipfs/js-ipfs-api/blob/c5e9d27cbb898fa8836125e8364c93f0adf6a78f/README.md#in-the-browser-through-script-tag), for example.